src/Tizen.BatteryMonitor/Tizen.BatteryMonitor/BatteryMonitor.cs

```diff
@@ -0,0 +1,206 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace Tizen.BatteryMonitor
```

Comment:
Do you have no group of module? like a Tizen.System

Reply:
BatteryMonitor comes under  system API group in Native. But it is maintained under BatteryMonitorFramework so we want to place it separately under namespace Tizen.BatteryMonitor

Similar to Sensor, as it also  comes under system API in native  but in TizenFX its namespace is Tizen.Sensor

Reply:
API does not grouped by ownership of management

Reply:
Sensor is enough complex to separate a new group, but BattaryMonitor is not.

Reply:
I see.
Will there be any conflict if we proceed with Tizen.BatteryMonitor ?

Reply:
maybe conflict with reviewer?

Reply:
Actually we had a discussion internally with Mr. O-HOON KWON regarding the name of the namespace, and we came conclusion to use Tizen.BatteryMonitor, that is why i proceeded with that name in previous commit.

Reply:
Do you have any issue to use Tizen.System.BatteryMonitor?

Reply:
Thank you for the reply. We are discussing on this, Will update you by tomorrow.

Reply:
Considering the API in view of real 3rd party developer. Battery API is already provided in Tizen.System namespace with Battery class.
Creating a new namespace Tizen.System.BatteryMonitor or Tizen.BatteryMonitor can create confusion.
So we decided to add this BatteryMonitor class in Tizen.System itself and further we would change class name to PowerUsage.
Now that it can be accessed by  "Tizen.System.PowerUsage"